xray/segment.go

```diff
@@ -227,34 +226,34 @@ func (seg *Segment) Close(err error) {
 	seg.Unlock()
 
 	if err != nil {
-		seg.AddError(err)
+		_ = seg.AddError(err) // Store only.
```

Comment:
why we add `_=` before calling the method?

Reply:
To comply with errcheck linter and be explicit on the error ignore. It makes it clear it is a vonluntary ignore vs a mistake.